README.md

```diff
@@ -1,35 +1,38 @@
 [![Build Status](https://travis-ci.org/splunk/splunk-sdk-javascript.svg?branch=master)](https://travis-ci.org/splunk/splunk-sdk-javascript)
-# The Splunk Software Development Kit for JavaScript
+# The Splunk Enterprise Software Development Kit for JavaScript
 
-#### Version 1.9.0
+#### Version 1.9.1
 
-The Splunk Software Development Kit (SDK) for JavaScript contains library code and
-examples designed to enable developers to build applications using Splunk and
-JavaScript. This SDK supports both server- and client-side JavaScript.
+The Splunk Enterprise Software Development Kit (SDK) for JavaScript contains library code and examples designed to enable developers to build applications using the Splunk platform and JavaScript. This SDK supports server-side and client-side JavaScript.
+
+For more information, see [Splunk Enterprise SDK for JavaScript](https://dev.splunk.com/enterprise/docs/devtools/javascript/sdk-javascript/) on the Splunk Developer Portal.
 
 ## Requirements
 
-* Node.js v.0.12, or v4 or later. The Splunk SDK for Javascript is tested with Node.js v.0.12, v4.2, and v10.0. 
-* Splunk Enterprise 6.3.0 or later, or Splunk Cloud. The Splunk SDK for Javascript is tested with Splunk Enterprise 7.0 and 7.2. 
+* Node.js v 0.12, or v4 or later
 
-## Installation
+    The Splunk Enterprise SDK for JavaScript was tested with Node.js v.0.12, v4.2, and v10.0. 
+
+* Splunk Enterprise 6.3.0 or later, or Splunk Cloud
 
-This section describes the basic steps for installing the Splunk SDK for JavaScript.
-For more detailed instructions and requirements, see the
-[Splunk Developer Portal][install].
+    The Splunk Enterprise SDK for JavaScript was tested with Splunk Enterprise 7.0 and 7.2. 
```

Comment:
~Splunk Enterprise 6.3.0 or later, or Splunk Cloud~
 Splunk Enterprise 8.2.0 or later, or Splunk Cloud

~The Splunk Enterprise SDK for JavaScript was tested with Splunk Enterprise 7.0 and 7.2.~
The Splunk Enterprise SDK for JavaScript was tested with Splunk Enterprise 8.0 and 8.2.

Reply:
README.md updated